src/article/ArticleItems.ts

```diff
@@ -0,0 +1,99 @@
+import { ArticleItemType } from './ArticleItemType';
+import { ItemArea } from './ItemArea';
+import { RichTextBlock, RichTextStyle } from './RichText';
+import { TextAlign } from './TextAlign';
+
+export type ItemAny = Item<ArticleItemType>;
+
+export interface Item<T extends ArticleItemType> {
+  id: string;
+  type: T;
+  area: Record<LayoutIdentifier, ItemArea>;
+  visible: Record<LayoutIdentifier, boolean>;
+  link?: Link;
+  commonParams: ItemCommonParamsMap[T];
+  layoutParams: Record<LayoutIdentifier, any>;
```

Comment:
isn't it  `layoutParams: Record<LayoutIdentifier, ItemLayoutParamsMap[T]>;`  instead any?